mvc_controller.cpp

```diff
@@ -331,19 +331,27 @@ void MvcController::ConditionallyEnableItems
         }
     else if(itembox)
         {
-        // WX !! wxWindowUpdateLocker doesn't seem to help much.
-        wxWindowUpdateLocker u(&control);
-        itembox->Clear();
-        // WX !! Append(wxArrayString const&) "may be much faster"
-        // according to wx online help, but that seems untrue: its
-        // implementation just uses a loop.
+        std::vector<wxString>& itembox_cache = itemboxes_cache_[name];
```

Comment:
I'd like to rename this variable to `cached_items` to make its relationship with `items` more clear.

I'd probably also use `auto&` here, but this is more subjective.